### PR TITLE
Move common Makefile targets to Makefile.common

### PR DESCRIPTION
**Pull request by rafaelvanoni**

### CODE REVIEW

File: pkg/log/log.go [hunk 1]
```diff
@@ -3,33 +3,21 @@ Package log provides support for logging to stdout and stderr.
 
 Log entries will be logged in the following format:
 
-    timestamp hostname tag[pid]: SEVERITY Message
```
Comment (tmjd) [inline pkg/log/log.go]: These changes seem unrelated to the Makefile.common change. What is the reason for the change?

Comment (rafaelvanoni) [reply pkg/log/log.go]: Lint picked these out. The `tag` variable is not being used.
